ocs_ci/ocs/dr/dr_workload.py

```diff
@@ -185,6 +212,144 @@ def deploy_workload(self):
 
         self.verify_workload_deployment()
 
+    def deploy_workloads_on_managed_clusters(
+        self, primary_cluster=True, secondary_cluster=None
+    ):
+        """
+        Deployment specific to busybox workload on both primary and secondary clusters
+
+        Args:
+            primary_cluster(bool) : True if apps needs to be deployed on primary cluster
+            secondary_cluster(bool) : True if apps needs to be deployed on secondary cluster
+
+        """
+        self._deploy_prereqs()
+        workload_namespaces = []
+
+        # By default, it deploys apps on primary cluster if not set to false
+        clusters = [self.preferred_primary_cluster]
+        if secondary_cluster:
+            clusters = [self.preferred_secondary_cluster]
+        if primary_cluster and secondary_cluster:
+            clusters.append(self.preferred_secondary_cluster)
+
+        for cluster in clusters:
+            # load workload-repo namespace.yaml
+            workload_ns_yaml_data = templating.load_yaml(self.namespace_yaml_file)
+            workload_ns_yaml_data["metadata"][
+                "name"
+            ] = helpers.create_unique_resource_name(
+                resource_type="namespace", resource_description="busybox-workloads"
+            )
+            templating.dump_data_to_temp_yaml(
+                workload_ns_yaml_data, self.namespace_yaml_file
+            )
+
+            # load placementrule
+            placementrule_yaml_data = templating.load_yaml(self.placementrule_yaml_file)
+            placementrule_yaml_data["metadata"][
+                "name"
+            ] = helpers.create_unique_resource_name(
+                resource_type="placementrule", resource_description="busybox"
+            )
+            templating.dump_data_to_temp_yaml(
+                placementrule_yaml_data, self.placementrule_yaml_file
+            )
+
+            # load drpc.yaml
+            drpc_yaml_data = templating.load_yaml(self.drpc_yaml_file)
+            drpc_yaml_data["metadata"]["name"] = helpers.create_unique_resource_name(
+                resource_type="drpc", resource_description="busybox"
+            )
+            drpc_yaml_data["spec"]["placementRef"]["name"] = placementrule_yaml_data[
+                "metadata"
+            ]["name"]
+            drpc_yaml_data["spec"]["preferredCluster"] = cluster
+            drpc_yaml_data["spec"]["drPolicyRef"]["name"] = self.dr_policy_name
+            templating.dump_data_to_temp_yaml(drpc_yaml_data, self.drpc_yaml_file)
+
+            # TODO
```

Comment:
Will it be done in this or another PR? If it's latter, we should create an issue to track it.